### PR TITLE
Enforce minimum df length in store

### PR DESCRIPTION
**Pull request by velramiir**
We agreed that a minimum size of the data sets is necessary for the column type heuristics to work correctly, we should therefore enforce it

### CODE REVIEW

File: tests/test_Store.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+import pandas as pd
+
+from shift_detector.precalculations.Store import InsufficientDataError, Store
+
+
+class TestStore(unittest.TestCase):
+
+    def test_min_data_size_is_enforced(self):
+        self.assertRaises(InsufficientDataError, Store, df1=pd.DataFrame(), df2=pd.DataFrame([0]))
```
Comment (Mupico) [inline tests/test_Store.py]: You only test that the assertion is raised if there is not enough data. I think you should also test that there is no assertion raised if there is enough data.

Newline at the end of the file is missing.

File: shift_detector/precalculations/Store.py [hunk 2]
```diff
@@ -36,3 +48,12 @@ def __getitem__(self, needed_preprocessing) -> DataFrame:
...
         self.preprocessings[needed_preprocessing] = preprocessing
         return preprocessing
+
+    @staticmethod
+    def verify_min_data_size(size):
+        if size < MIN_DATA_SIZE:
+            raise InsufficientDataError('The input data is insufficient for the column type heuristics to work. Only '
+                                        '{actual} row(s) were passed. Please pass at least {expected} rows.'.format(
+                                            actual=size,
+                                            expected=MIN_DATA_SIZE),
```
Comment (Mupico) [inline shift_detector/precalculations/Store.py]: indent seems to be a bit broken here

File: tests/precalculations/test_text_metadata.py [hunk 3]
```diff
@@ -9,115 +9,169 @@
...
     def setUp(self):
         poems = [
             'Tell me not, in mournful numbers,\nLife is but an empty dream!\nFor the soul is dead that slumbers,\nAnd things are not what they seem.',
-            'Life is real! Life is earnest!\nAnd the grave is not its goal;\nDust thou art, to dust returnest,\nWas not spoken of the soul.'
-        ]
-        phrases = [
-            'Front-line leading edge website',
-            'Upgradable upward-trending software'
-        ]
+            'Life is real! Life is earnest!\nAnd the grave is not its goal;\nDust thou art, to dust returnest,\nWas not spoken of the soul.',
```
Comment (Mupico) [inline tests/precalculations/test_text_metadata.py]: potentially to long but thats currently part of discussion

File: shift_detector/precalculations/Store.py (hunk 2 above)
Comment (velramiir) [reply shift_detector/precalculations/Store.py]: This indent was intended to separate between the params of the error and the format() method. Open to better suggestions though

Comment (Mupico) [reply shift_detector/precalculations/Store.py]: ah okay now I get it.

**Pull request merged**